@l10n/ja/docs/concepts/tokens/decentralized-exchange/permissioned-domains.md

```diff
@@ -0,0 +1,39 @@
+---
+seo:
+    description: 許可型ドメインの定義と詳細
+labels:
+  - コンプライアンス
+  - 許可型ドメイン
+---
+# 許可型ドメイン
+
+許可型ドメインは、XRP Ledgerブロックチェーンのより広いエコシステム内の制御された環境です。ドメインは何もしませんが、許可型DEXや貸付プロトコルなどの機能は、ドメインを使用してアクセスを制限するため、従来の金融機関は、様々なコンプライアンスルールに従って、チェーン上でサービスを提供できます。
+
+ドメインの唯一の設定可能なルールは、[資格情報][]のセットです。将来のAmendmentは、金融機関が業務を行う地域の法律に従ってコンプライアンスを維持するために必要な制限をカバーするために、新しいおよび異なるタイプのルールを追加することができます。
+
+誰でもドメインを定義できます。その人はそのドメインの所有者になり、設定を更新したり削除したりできます。ドメインの数の制限は、所有者の準備金要件に対して1つのアイテムとしてカウントされるため、レジャーに存在できるドメインの数の制限は、準備金要件のみです。
+
+_([PermissionedDomains amendment][]が必要です {% not-enabled /%})_
+
+{% admonition type="info" name="注記" %}
+[Credentials amendment][]も必要です。[PermissionedDomains amendment][]が有効でない場合、PermissionedDomainSetトランザクションは無効とみなされます。
+{% /admonition %}
+
+## 許可された資格情報
+
+許可型ドメインには、アクセスを許可する1から10個の資格情報のセットがあります。リストの各項目は、発行者と資格情報の種類によって資格情報を説明します。_少なくとも1つ_ 一致する資格情報を保持するアカウントは、自動的にドメインにアクセスできます。
+
+ドメインは、資格情報と制限されたリソースの間の抽象化レイヤーとして機能します。ドメインの所有者は、制限されたリソース自体を変更せずに、必要な資格情報のリストを更新できます。
+
+[{% inline-svg file="../../../img/permissioned-domain.svg" /%}](../../../img/permissioned-domain.svg "図: 許可型DEXは、IDによって許可型ドメインを指します。ドメインの許可された資格情報は、アクセスを取得するための3つの可能な資格情報を説明します")
```

Comment:
I realized that the source file for this diagram is missing. I'll open a separate PR to add the Umlet file so you can translate it.

Reply:
I will keep this as a future task.